cmd/installations/create.go

```diff
@@ -326,6 +340,7 @@ func (o *createOpts) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.name, "name", "my-installation", "name of the installation")
 	fs.BoolVar(&o.renderSchemaInfo, "render-schema-info", true, "render schema information of the component's imports and exports as comments into the installation")
 	fs.StringVar(&o.blueprintResourceName, "blueprint-resource-name", "", "name of the blueprint resource in the component descriptor (optional if only one blueprint resource is specified in the component descriptor)")
+	fs.StringVarP(&o.outputPath, "output", "o", "", "file path for the resulting installation yaml")
```

Comment:
the parameter should be named "output-file" to be more coherent with the rest of the CLI?